pages/full.js

```diff
@@ -50,6 +50,77 @@ const Full = () => (
+              fill="#FF6AB9"
+            />
+          </svg>
+          <Heading variant="h2" className={styles.featureHeading}>
```

Comment:
nitpicking here but you can use the ENUM`Heading.VARIANT.h2` for the variant type so you don't have to pass a string

Reply:
Ah good catch :+1: